packages/remix-server-runtime/cookies.ts

```diff
@@ -245,3 +248,14 @@ function myUnescape(value: string): string {
   }
   return result;
 }
+
+function warnOnceAboutExpiresCookie(name: string, expires?: Date) {
+  warnOnce(
+    !(expires instanceof Date),
```

Comment:
It would lower the cognitive load if we just check if `expires` is set

```suggestion
    !expires,
```